TourMate/TourMate/Frontend/Plan/Views/PlansViews/PlansListView.swift

```diff
@@ -24,7 +24,7 @@ struct PlansListView: View {
     }
 
     var body: some View {
-        LazyVStack {
+        ScrollView {
```

Comment:
I don't think the `ScrollView` here isn't active because of the parent `ScrollView` in `TripView`

Reply:
Can consider changing it into a `VStack` instead then

Reply:
oh ok